custom_components/frigate/camera.py

```diff
@@ -288,6 +302,16 @@ async def favorite_event(self, event_id: str, favorite: bool) -> None:
         """Favorite an event."""
         await self._client.async_retain(event_id, favorite)
 
+    async def ptz(self, service: str, action: str) -> None:
+        """Run PTZ command."""
+        await async_publish(
+            self.hass,
+            self._ptz_topic,
+            f"{service}-{action}" if service == "preset" else f"{service}_{action}",
```

Comment:
This is strange, is there a good reason for `-` vs `_` in the backend?

Reply:
I can't remember for sure, but I think _ is more common in preset names that are set by some cameras

Reply:
Is the backend not just splitting this by either `-` or `_`. I guess the question is: Can we just pick one delimiter?

Reply:
It is, the original reason was just to be careful that the preset name wasn't split. I will put up a PR to align those because I agree it is somewhat arbitrary

Reply:
Aha, SG. So pick one, and only split on that one once from the left to allow the preset to still contain whatever it wants?